README.MD

```diff
@@ -25,7 +25,47 @@ Camino-Messenger-Bot is designed to facilitate communication through various mes
 
 This application depends on the [camino-messenger-protocol](https://github.com/chain4travel/camino-messenger-protocol), which is a separate repository. The protocol defines the protobuf messages and services used for communication between the gRPC server and client.
 
-Currently [release-9](https://github.com/chain4travel/camino-messenger-protocol/releases/tag/release-9) of Camino Messenger Protocol is used.
+Currently [release-10](https://github.com/chain4travel/camino-messenger-protocol/releases/tag/release-10) of Camino Messenger Protocol is used.
+
+## Camino Messenger Account (CM Account)
+
+To run the camino-bot-messenger as intended you will need a **CM Account** with a **bot** added to it and **registered services**.
+
+### Creating the CM Account
+If you don't already have a **CM Account**, for development purposes, a CM Account can be created via [camino messenger contracts](https://github.com/chain4travel/camino-messenger-contracts) `hardhat` tool or via [suite](https://suite.camino.network/create) as explained [here](https://docs.camino.network/partners/partner-config/#step-by-step-configuration-guide).
```

Comment:
> `... via [suite](https://suite.camino.network/create) ...`
This part is a little bit misleading. Because it links to the _create_ page of the wallet, when someone clicks on it the wallet generations page opens. We should change this to only the suite link, without the `/create` at the end.

Reply:
Done